gc/base/GCExtensionsBase.hpp

```diff
@@ -580,7 +580,9 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 	float excessiveGCFreeSizeRatio;
 
 	MM_Heap* heap;
+#if	defined(OMR_GC_SPARSE_HEAP_ALLOCATION)
```

Comment:
I realized that spacing is not good.
@babsingh Should I fix right away or to wait for jenkins testing to complete?